api-reference/10 UI Components/dxOverlay/1 Configuration/hideOnParentScroll.md

```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to hide the {WidgetName} when users scroll one of the {WidgetName}'s parent elements. 
```

Comment:
Is it necessary to mention the WidgetName twice? For example, can we update this to the following?

`Specifies whether to hide the {WidgetName} when users scroll one of its parent elements.`

Reply:
Sounds reasonable, thanks

Reply:
Thank you